### DISCUSSION TITLE
How to see the debug logs on ubuntu

### DISCUSSION THREAD

**Discussion by rodrigo081089** [Chia-Network/chia-blockchain#4634] [Q&A]
I had chia installed on windows 10 but I had the This should be below 5 seconds to minimize risk of losing rewards problem accesing my NAS. So I wanna use ubuntu instead, I hace chia plotting on ubuntu and I added my keys but I dont know how to see if chia is farming my plots or not, and the log directory is empty. I wanna know if im below 5 seconds on ubuntu or not.

Thanks

**Comment by lvcivs** [comment]
The log is in `~/.chia/mainnet/log/debug.log`. If you don't have that file, then your node has not been started up.

**Reply by rodrigo081089** [reply]
How do I start the node?

**Reply by lvcivs** [reply]
Depends on whether you installed from git or the .deb installer and also whether you want to run the GUI or from the command line.

**Reply by rodrigo081089** [reply]
Im running on cli with the git

**Reply by lvcivs** [reply]
`cd chia-blockchain`
`. ./activate`
`chia start farmer`

**Comment by rodrigo081089** [comment]
Thanks, and can I keep windows and ubuntu farming the same plots at the same time so I can access faster to the plots and dont have the 30 seconds problem?

**Reply by lvcivs** [reply]
It is not recommended to run two nodes at the same time in the same LAN.

**Reply by rodrigo081089** [reply]
On different network, is that ok?
